bindings/matrix-sdk-crypto-nodejs/package.json

```
@@ -21,7 +21,8 @@
         "node": ">= 14"
     },
     "scripts": {
-        "build": "napi build --platform --release --strip",
+        "release-build": "napi build --platform --release --strip",
+        "build": "napi build --platform",
```

I bet we could still keep the `--release` option?

```suggestion
        "build": "napi build --platform --release",
```

---

sure, it is slower though and is it is mainly/only used (here) to build the debug-version for the upload, I do want it to not remove the debug symbols (which `--release` does, doesn't it)...

---

This is only meant for testing / debugging, right? When we actually publish a release I agree we should use `--release`, but always using release builds for CI is unusual and I don't think we should have been doing it in the first place.

---

@Hywan , @jplatte do you want to discuss this to conclusion on this PR now, or can I get an approving review from either of you to merge as is and track upcoming failures?

---

Well, @Hywan self-assigned this, so I've been holding off the approval.